vllm/engine/metrics.py

```diff
@@ -232,6 +254,22 @@ def __init__(self, labelnames: List[str], vllm_config: VllmConfig):
             labelnames=labelnames,
             buckets=build_1_2_5_buckets(max_model_len),
         )
+        self.gauge_max_token_capacity_request = self._gauge_cls(
```

Comment:
This needs a better name.

Reply:
This metric captures the maximum tokens that can be processed by the model server in total at its maximum batch size ([ref](https://docs.google.com/document/d/1SpSp1E6moa4HSrJnS4x3NpLuj88sMXr2tbofKlzTZpk/edit?resourcekey=0-ob5dR-AJxLQ5SvPlA4rdsg&tab=t.0#heading=h.8lsb0mki35at
)). How about `max_token_capacity_per_batch`?